source/speech/speech.py

```diff
@@ -1861,6 +1860,127 @@ def getPropertiesSpeech(  # noqa: C901
+	"""Gets the speech sequence for a shortcuts string containing one or more shortcuts.
+	@param keyboardShortcutsStr: the shortcuts string.
+	"""
+	
+	SHORTCUT_KEY_LIST_SEPARATOR = '  '
+	seq = []
+	if not keyboardShortcutsStr:
+		return seq
+	try:
+		for shortcutKeyStr in keyboardShortcutsStr.split(SHORTCUT_KEY_LIST_SEPARATOR):
+			seq.extend(_getKeyboardShortcutSpeech(shortcutKeyStr))
+			seq.append(SHORTCUT_KEY_LIST_SEPARATOR)
+		seq.pop()  # Remove last SHORTCUT_KEY_LIST_SEPARATOR in the sequence
+	except Exception:
+		log.warning(
+			f'Error parsing keyboard shortcut "{keyboardShortcutsStr}", reporting the string as a fallback.',
+			exc_info=True,
+		)
+		return [keyboardShortcutsStr]
+
+	# Merge consecutive strings in the list when possible
+	seqOut = []
+	for item in seq:
+		if len(seqOut) > 0 and isinstance(seqOut[-1], str) and isinstance(item, str):
+			seqOut[-1] = seqOut[-1] + item
+		else:
+			seqOut.append(item)
+
+	return seqOut
+
+
+def _getKeyboardShortcutSpeech(keyboardShortcut: str) -> SpeechSequence:
+	"""Gets the speech sequence for a single shortcut string.
+	@param keyboardShortcut: the shortcuts string.
+	"""
+
+	if ', ' in keyboardShortcut:
+		keyList, separators = _splitSequentialShortcut(keyboardShortcut)
+	elif '+' in keyboardShortcut and len(keyboardShortcut) > 1:
+		keyList, separator = _splitShortcut(keyboardShortcut)
+		separators = [separator] * (len(keyList) - 1)
+	else:
+		return _getKeySpeech(keyboardShortcut)
+
+	seq = []
+	for key, sep in zip(keyList[:-1], separators):
+		seq.extend(_getKeySpeech(key))
+		seq.append(sep)
+	seq.extend(_getKeySpeech(keyList[-1]))
+	return seq
+
+
+def _getKeySpeech(key: str) -> SpeechSequence:
+	"""Gets the speech sequence for a string describing a key.
+	@param key: the key string.
+	"""
+
+	if len(key) > 1:
+		return [key]
+	locale = getCurrentLanguage()
+	keySymbol = characterProcessing.processSpeechSymbol(locale, key)
+	if keySymbol != key:
+		return [keySymbol]
+	return [
+		CharacterModeCommand(True),
+		key,
+		CharacterModeCommand(False),
+	]
+
+
+def _splitShortcut(shortcut: str) -> Tuple[List[str], str]:
+	"""Splits a string representing a shortcut key combination.
+	@param shortcut: the shortcut to split.
+		It may be of the form "NVDA+R" or "NVDA + R", i.e. key names separated by "+" symbol with or without
+		space around it.
+	@return: 2-tuple containing the list of the keys and the separator used between them.
+		E.g. (['NVDA', 'R'], ' + ')
+	"""
+	
+	if ' + ' in shortcut:
+		separator = ' + '
+	elif '+' in shortcut:
+		separator = '+'
+	else:
+		raise RuntimeError(f'The shortcut "{shortcut}" needs to contain a "+" symbol.')
+	if shortcut[-1] == '+':  # E.g. "Ctrl+Shift++"
+		keyList = shortcut[:-1].split(separator)
+		keyList[-1] = keyList[-1] + '+'
+	else:
+		keyList = shortcut.split(separator)
+	return keyList, separator
+
+
+def _splitSequentialShortcut(shortcut: str) -> Tuple[List[str], List[str]]:

```

Comment:
Not at all. The comma and the space do not have the same meaning in Word's sequential shortcuts. The comma separates the tooltips seen visually in the ribbon. On each control of the ribbon, the tooltip is displayed with one or two letters. When there are two letters, they are separated by a space without comma.

Let's take an example:
In my French Word, in the "Accueil" (Home) ribbon, there is a button called "Effacer la mise en forme" (clear formatting). 
1. When I press Alt, all the tooltips in the upper ribbon appear visually; a "L" appears on "Accueil"
2. If I arrow left/right to "Accueil" ribbon, I can hear "Alt, L" when it is selected. That means that I can press "Alt, L" to get directly to it next time when the ribbon is not yet selected.
3. I select this ribbon
4. In the lower ribbon the tooltips are now displayed for all the controls. Among which "MF" for the button "Effacer la mise en forme". That means that if I press M then F at this point, the button will be activated. Or said otherwise, if I want to activate the button when the ribbon is not selected, I have to press Alt, then L then M then F.

So pressing "Alt, F" selects something and pressing "Alt, L, M F" selects something. But "Alt, L, M" does not do anything more than "Alt, L" since another character is expected. Hence the space between the M and the F.